src/services/shared.ts

```diff
@@ -32,5 +32,10 @@ export const logRocketConsole = (message: string, ...props: any[]) => {
 export const FAILED_TO_FETCH = 'failed to fetch';
 export const RETRY_REASONS = [FAILED_TO_FETCH];
 
+export const checkErrorMessage = (
+    checkingFor: string,
+    message?: string | null | undefined
+) => (message ? message.toLowerCase().includes(checkingFor) : false);
+
```

Comment:
Quick helper function to make this check consistent. Eventually we might just make a `includesIgnoreCase` function or something like that. But wanted to have this function start by the variables it will normally be used to test.